src/settings/WatchManager.ts

```diff
@@ -18,11 +18,10 @@ import { SettingLevel } from "./SettingLevel";
 
 export type CallbackFn = (changedInRoomId: string, atLevel: SettingLevel, newValAtLevel: any) => void;
 
-const IRRELEVANT_ROOM = Symbol("any room");
+const IRRELEVANT_ROOM = String(null);
```

Comment:
why not `IRRELEVANT_ROOM: string = null`?

Reply:
Because it'll upset TS if we ever turn on strict null checking :P

Reply:
But will do so anyhow, for consistency with write

Reply:
I imagine we won't turn on strict null checking, mostly because exactly zero projects in the company have strict null checking.